src/api/mvc/tables/main.go

```diff
@@ -0,0 +1,306 @@
+package tables
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"time"
+
+	"api/contexts"
+	"api/models/constants"
+	"api/models/dtos"
+	"api/models/dtos/errors"
+	"api/models/indexes"
+	"api/mvc"
+	esRepo "api/repositories/elasticsearch"
+	"api/utils"
+
+	"github.com/labstack/echo"
+	"github.com/mitchellh/mapstructure"
+)
+
+func CreateTable(c echo.Context) error {
+	fmt.Printf("[%s] - CreateTable hit!\n", time.Now())
+
+	decoder := json.NewDecoder(c.Request().Body)
+	var t dtos.CreateTableRequestDto
+	err := decoder.Decode(&t)
+	if err != nil {
+		return c.JSON(http.StatusBadRequest, map[string]interface{}{
+			"error": err,
+		})
+	}
+
+	log.Println("Incoming:")
```

Comment:
Has it been left intentionally?

Reply:
Not really - there are better ways to log what i intended to see here in production -- will remove this and revisit this when need be